src/controllers/FikaSendItemController.ts

```diff
@@ -100,6 +100,12 @@ export class FikaSendItemController {
         const profiles = this.saveServer.getProfiles();
 
         for (const profile of Object.values(profiles)) {
+            //Uninitialized profiles can cause this to error out, skip these.
+            if (profile.characters.pmc.Info == null)
+            {
+                continue;
+            }
```

Comment:
```suggestion
            if (profile.characters.pmc.Info === null)
            {
                continue;
            }
```

I think `===` would be better.